docs/developers/debugging/releases.md

```diff
@@ -84,3 +84,50 @@ Other tips:
   Release mode `main-dist` package above once, which will drop binaries in the
   `iree-install` directory. Then build the `py-runtime-pkg` or equiv and
   iterate further in the build directory. Ditto for TF/XLA/etc.
+
+## Testing releases on your fork
+
+To avoid interrupting the regular releases published on the IREE github, you
+can test any changes to the release process on your own fork.  Some setup is
+required before these github actions will work on your fork and development
+branch.
+
+To run
+[`schedule_snapshot_release.yml`](https://github.com/google/iree/blob/main/.github/workflows/schedule_snapshot_release.yml),
+comment out
+[this line](https://github.com/google/iree/blob/main/.github/workflows/schedule_snapshot_release.yml#L14):
```

Comment:
I include some links to specific lines which seems like it could become outdated quickly, but I am not sure what the best way to handle this is

Reply:
Thanks for the screenshot!